src/Microsoft.ManagedZLib/src/TreeDesc.cs

```diff
@@ -0,0 +1,310 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.ManagedZLib;
+
+public class TreeDesc
+{
+    public CtData[] _dynamicTree;
+    public int _maxCode;
+    public StaticTreesDesc _StaticTreeDesc;
```

Comment:
Fields should start with lowercase:
```suggestion
    public StaticTreesDesc _staticTreeDesc;
```